generators/client/index.js

```diff
@@ -1,199 +1,213 @@
-        'nock': '9.0.9',
-        'nodemon': '1.11.0',
-        'nyc': '10.1.2',
-        'postcss-cssnext': '2.10.0',
-        'postcss-import': '9.1.0',
-        'postcss-loader': '1.3.3',
-        'prettier': '1.7.0',
-        'react-addons-test-utils': '15.6.2',
-        'react-hot-loader': '3.0.0-beta.6',
-        'redux-mock-store': '1.2.2',
-        'sinon': '2.0.0',
-        'style-loader': '0.16.1',
-        'stylelint-config-standard': '16.0.0',
-        'stylelint': '7.10.0',
-        'supertest': '3.0.0',
-        'url-loader': '0.5.8',
-        'webpack-dev-server': '2.9.4',
-        'webpack': '3.8.1',
-        'webpack-bundle-analyzer': '2.9.0'
-      },
+      devDependencies: {
+        "babel-cli": "6.24.1",
+        "babel-core": "6.24.1",
+        "babel-eslint": "7.2.3",
+        "babel-istanbul": "0.12.2",
+        "babel-jest": "19.0.0",
+        "babel-loader": "6.4.1",
+        "babel-plugin-transform-runtime": "6.23.0",
+        "babel-polyfill": "6.23.0",
+        "babel-plugin-lodash": "3.2.11",
+        "babel-preset-env": "1.6.1",
+        "babel-preset-es2015": "6.24.1",
+        "babel-preset-react-hmre": "1.1.1",
+        "babel-preset-react": "6.24.1",
+        "babel-preset-stage-0": "6.24.1",
+        chai: "3.5.0",
+        chromedriver: "2.28.0",
+        "clean-webpack-plugin": "0.1.17",
+        "css-loader": "0.27.3",
+        enzyme: "3.1.0",
+        "enzyme-adapter-react-16": "1.0.0",
+        "eslint-config-airbnb": "15.1.0",
+        "eslint-config-prettier": "2.3.0",
+        "eslint-loader": "1.9.0",
+        "eslint-plugin-import": "2.7.0",
+        "eslint-plugin-jsx-a11y": "5.1.1",
+        "eslint-plugin-prettier": "2.1.2",
+        "eslint-plugin-react": "7.1.0",
+        eslint: "4.3.0",
+        expect: "1.20.2",
+        "extract-text-webpack-plugin": "3.0.2",
+        "favicons-webpack-plugin": "0.0.7",
+        "file-loader": "0.10.1",
+        "html-webpack-plugin": "2.30.1",
+        istanbul: "0.4.5",
+        "jest-css-modules": "1.1.0",
+        jest: "21.2.1",
+        "jsdom-global": "2.1.1",
+        jsdom: "9.12.0",
+        jshint: "2.9.4",
+        "json-loader": "0.5.4",
+        mocha: "3.2.0",
+        nock: "9.0.9",
+        nodemon: "1.14.7",
+        nyc: "10.1.2",
+        plop: "1.9.1",
+        "postcss-cssnext": "2.10.0",
+        "postcss-import": "9.1.0",
+        "postcss-loader": "1.3.3",
+        prettier: "1.7.0",
+        "react-addons-test-utils": "15.6.2",
+        "react-hot-loader": "3.0.0-beta.6",
+        "redux-mock-store": "1.2.2",
+        sinon: "2.0.0",
+        "style-loader": "0.16.1",
+        "stylelint-config-standard": "16.0.0",
+        stylelint: "7.10.0",
+        supertest: "3.0.0",
+        "url-loader": "0.5.8",
+        "webpack-dev-server": "2.9.4",
+        webpack: "3.10.0",
+        "webpack-bundle-analyzer": "2.9.2"
+      }
     };
 
-    const packageJsonPath = 'package.json'
+    const packageJsonPath = "package.json";
```

Comment:
why always double quotes instead of single quotes ?

Reply:
it isnt prettified yet